### PR TITLE
Switch from cloudbees to bintray

### PR DESCRIPTION
**Pull request by ThomDietrich**
@BClark09 

Signed-off-by: Thomas Dietrich <Thomas.Dietrich@tu-ilmenau.de>

### CODE REVIEW

File: openhabian-setup.sh [hunk 1]
```diff
@@ -234,11 +234,8 @@ java_webupd8_install() {
 openhab2_addrepo() {
   echo -n "[openHABian] Adding openHAB 2 Snapshot repositories to sources.list.d... "
   rm -f /etc/apt/sources.list.d/openhab2.list
-  echo "deb https://openhab.ci.cloudbees.com/job/openHAB-Distribution/ws/distributions/openhab-offline/target/apt-repo/ /" >> /etc/apt/sources.list.d/openhab2.list
-  echo "deb https://openhab.ci.cloudbees.com/job/openHAB-Distribution/ws/distributions/openhab-online/target/apt-repo/ /" >> /etc/apt/sources.list.d/openhab2.list
-  # out of an unclear reason, this is not working:
-  #cond_redirect wget --retry-connrefused --tries=50 -O - http://www.openhab.org/keys/public-key-snapshots.asc | apt-key add -
-  cond_redirect wget --retry-connrefused --tries=50 -O openhab-key.asc http://www.openhab.org/keys/public-key-snapshots.asc
+  echo "deb http://dl.bintray.com/openhab/apt-repo2 unstable main" >> /etc/apt/sources.list.d/openhab2.list
```
Comment (BClark09) [inline openhabian-setup.sh]: You won't need the rm -f if you remove a ">" from ">>".

File: openhabian-setup.sh [hunk 2]
```diff
@@ -234,11 +234,8 @@ java_webupd8_install() {
...
+  cond_redirect wget --retry-connrefused --tries=50 -O openhab-key.asc 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab'
```
Comment (BClark09) [inline openhabian-setup.sh]: Hopefully, as a distribution centre Bintray will be more stable ;)

Comment (ThomDietrich) [inline openhabian-setup.sh]: This was really an annoyance...

File: openhabian-setup.sh (hunk 1 above)
Comment (ThomDietrich) [inline openhabian-setup.sh]: You are right of course. A leftover from when two repo links were needed ;)

**Comment by ThomDietrich** [comment]
Thanks @BClark09

**Pull request merged**